.devcontainer/devcontainer.json

```diff
@@ -33,6 +33,11 @@
             "label": "Web VNC"
         }
     },
+    "remoteEnv": {
+        "CC": "clang-20",
+        "CXX": "clang++-20",
```

Comment:
should this be added in the ladybird feature that determines the llvm_version installed instead of here at the top level?